phpcs.xml.dist

```diff
@@ -10,13 +10,12 @@
   <arg name="cache" value="build/.phpcs-cache"/>
   <arg name="parallel" value="10"/>
 
-  <file>.</file>
+  <file>src</file>
+  <file>tests</file>
 
   <!-- Danger! Exclude patterns apply to the full file path, including parent directories of the current repository. -->
   <!-- Don't exclude common directory names like `build`, which will fail on Travis CI because of /home/travis/build/acquia/<project>. -->
   <!-- @see https://github.com/squizlabs/PHP_CodeSniffer/issues/981 -->
-  <exclude-pattern>var/</exclude-pattern>
```

Comment:
I discovered a PhpStorm bug that causes it to silently fail to run PHPCS if you apply this particular exclude pattern. It's truly bizarre.

Changing the configuration to only scan source and test directories allows us to work around the bug and is more performant to boot.